src/base/opt/algsig.rs

```diff
@@ -377,8 +377,8 @@ impl<'a, Variant, Octs> IntoIterator for &'a Understood<Variant, Octs>
 where
     Octs: AsRef<[u8]> + ?Sized
 {
-    type Item = SecAlg;
-    type IntoIter = SecAlgsIter<'a>;
+    type Item = SecurityAlgorithm;
+    type IntoIter = SecurityAlgorithmsIter<'a>;
```

Comment:
Nitpicky, but I would write this in singular form.

Reply:
Done